python/pyspark/sql/functions/builtin.py

```diff
@@ -15098,6 +15098,38 @@ def from_json(
     return _invoke_function("from_json", _to_java_column(col), schema, _options_to_str(options))
 
 
+@_try_remote_functions
+def parse_json(
```

Comment:
should be listed in `python/docs/source/reference/pyspark.sql/functions.rst`

Reply:
done.